src/components/ComponentDetails/EventTable/index.tsx

```diff
@@ -54,6 +54,13 @@ const EventTable: React.FC<EventTableProps> = ({ eventData }) => {
       <ic-typography variant="h3" apply-vertical-margins>
         <h4>Events</h4>
       </ic-typography>
+      <ic-typography apply-vertical-margins>
+        <p>
+          All components also accept native events supported by the DOM, such as{" "}
+          <code className="language-text">onClick</code> and{" "}
+          <code className="language-text">onSubmit</code>.
```

Comment:
i feel like "onSubmit" is not the best example, as this would only apply if the component was used on a form. maybe use a mouse or keyboard event instead??